src/com.frostycoolslug.goxlr-utility.sdPlugin/app/display-volume.js

```diff
@@ -58,19 +55,21 @@ class VolumeMonitor {
 
     #event_handle = () => {};
 
-    constructor(context, serial, mix, channel_a, channel_b) {
+    constructor(context, serial, mix, channel_a, channel_b, hide_name) {
         this.context = context;
         this.serial = serial;
         this.mix = mix;
         this.channel_a = channel_a;
         this.channel_b = channel_b;
-
+        this.hide_name = hide_name;
         this.monitor_a = `/mixers/${serial}/levels/volumes/${channel_a}`;
         this.monitor_b = `/mixers/${serial}/levels/submix/inputs/${channel_b}/volume`;
 
         // Monitor for Submix Enable / Disable...
         this.monitor_submix = `/mixers/${serial}/levels/submix`;
-        this.device = `/mixers/${serial}`
+        this.device = `/mixers/${channel_b}`
```

Comment:
Is this an intentional change? The device path is checked to handle situations where the GoXLR has been unplugged (so a REMOVE notification on the `/mixers/${serial}` path), I think this change will break that.

Reply:
no, i'm not sure how that happened